### ISSUE TITLE
Universal Swatch Slots - No Swatches Appearing

### ISSUE THREAD

**Issue by dazimmermann** [Loupimo/UniversalSwatchSlots#11]
Follow up to Discord Mods - help-using-mods post  [https://discord.com/channels/555424930502541343/1315924630959292456](https://discord.com/channels/555424930502541343/1315924630959292456) 

You requested my log and config file.  The SMM log is attached.

[SMMDebug-2024-12-10-14-13-29.zip](https://github.com/user-attachments/files/18080913/SMMDebug-2024-12-10-14-13-29.zip)

GitHub won't let me upload .cfg extensions so I've zipped it, but it's whatever you initially write to the config file when installing the mod and the game is restarted. 

[UniversalSwatchSlots.zip](https://github.com/user-attachments/files/18080936/UniversalSwatchSlots.zip)

Just a reminder that I DID have More Swatch Slots by Kyrium installed previously.

Thanks for your help.
Reddit/Discord: wivaca

**Comment by dazimmermann** [comment]
Thanks, I'll give it a try next time I sit down to play.

**Comment by Loupimo** [comment]
Yes I'm planning to do a custom UI as the configuration can be really long and ugly in the ingame menu (I'm pretty sure SML one was not designed for such long config file ^^). I have planned to use expandable lists in order to make it smaller :) (and I hope more optimized)
![image](https://github.com/user-attachments/assets/d0cece78-6143-41fe-9902-5db9526fde72)
UI is working fine but now I need to implement the logic (using SML one as much as possible).

Yes as you said I'm new to modding that's why I wanted to let to the user to activate it only to the desired sessions without messing up with every save in cas of hard bug / corrupt. I may plan to add a default palette (set to english) if none is selected now it is working a bit better :)
